configs/ote_custom_classification/mobilenet_v3_large_075/template.yaml

```diff
@@ -15,6 +15,7 @@ framework: OTEClassification v1.2.3
 entrypoints:
   base: torchreid.integration.sc.task.OTEClassificationTask
```

Comment:
Entry point should be changed to `OTEClassificationTrainTask`

Reply:
Done